using/messaging.md

```diff
@@ -65,6 +67,18 @@ You'll see your message printed below messages from others that came before it:
 ~your-urbit= Hello, world!
 ```
 
+### Subscribing to a Channel in Landscape
+
+In Landscape, you can always navigate to a channel by opening the menu, or using the shortcut `cmd-k`, then typing `go ~dopzod/urbit-help`. However, if you want to see messages in your inbox, you'll need to subscribe.
+
+Let's try. From the Talk terminal, run:
+
+```
+;source %inbox ~dopzod/urbit-help
```

Comment:
`;join` is shorthand for `;source %inbox`, and also sets presence, which this does not.

Reply:
Oh, cool – it looks like `;join` subscribes you in Landscape now. IIRC, that wasn't the case before, and when I asked @loganallenc about it this morning, he said you can follow up `;join` with `;source %inbox` to subscribe in Landscape. I thought I'd also tested it to confirm, but looks like I was wrong – `;join` does indeed subscribe you in Landscape. I'll change this.

Reply:
Honestly the real fix is to have a "join" button in Landscape that just sends a `%source` action to the local hall, which is all that `;join` [does](https://github.com/urbit/arvo/blob/master/app/talk.hoon#L1199) anyway. This used to be in at some point, not sure why it was removed.

_cries in hall_

Reply:
@Fang- I want that feature back as well – it was removed for _some_ reason